registration/tests/utils.py

```diff
@@ -85,12 +106,18 @@ def fill_form(self, form_data):
                 self.client.execute_script("document.getElementById('{}').type='text'".format(id_elem))
 
             if not element.get_attribute('readonly') and not element.get_attribute('type') == 'hidden':
+                element.click()
                 element.clear()
                 if value:
+                    time.sleep(.25)
```

Comment:
Could you please add a comment here, explaining why we're doing this?

Reply:
:+1: